### PR TITLE
Temporary pixel for error page

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/1201807753394693/1208602127384122/f

### Description
Add a pixel that is fired when error page is displayed

### Steps to test this PR

_Error page pixel_
- [x] Instal from branch
- [x] Put phone in airplane mode
- [x] Visit a site and check error page is shown
- [x] Go to logcat and filter by `m_errorpageshown`
- [x] Check `Pixel failed: 1 m_errorpageshown` is logged
- [x] Disable airplane mode
- [x] Close the app
- [x] Open the app again
- [x] Check `Pixel sent: 1 m_errorpageshown` is logged

_Error page pixel disabled_
- [x] Instal from branch
- [x] Go to _Settings > Feature Flag Inventory_ and disable `errorPagePixel` FF
- [x] Put phone in airplane mode
- [x] Visit a site and check error page is shown
- [x] Go to logcat and filter by `m_errorpageshown`
- [x] Check `Pixel failed: 1 m_errorpageshown` is **not** logged

### CODE REVIEW

**Comment by nalcalag** [comment]
* **#5193** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5193?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nalcalag and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/browser/errors/ErrorPagePixelFeature.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.duckduckgo.feature.toggles.api.Toggle
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "errorPagePixel",
+)
+interface ErrorPagePixelFeature {
+    @Toggle.DefaultValue(true)
+    fun self(): Toggle
+}
```
Comment (aitorvs) [inline app/src/main/java/com/duckduckgo/app/browser/errors/ErrorPagePixelFeature.kt:30]: Why not putting the feature flag under `AndroidBrowserConfigFeature` instead? It's better than having a top level feature.

Comment (nalcalag) [reply app/src/main/java/com/duckduckgo/app/browser/errors/ErrorPagePixelFeature.kt:30]: Good idea, I'll change it

**Review by joshliebe — APPROVED** [review]
Tested and works as expected, good job 🙂

**Pull request merged**